multiregion/docs/multiregion.rst

```diff
@@ -48,6 +48,14 @@ participate in the ISR list.
 
 |Follower_Fetching|
 
+``Observer Promotion`` is the process whereby an observer is promoted into the
+ISR in certain degraded scenarios. This behaviour is controlled by the
+``observerPromotionPolicy`` field in a topic's replica placement policy. It can
+have values:
+
+- under-min-isr: observers will be promoted if the isr size drops below the topic's min.insync.replicas configuration.
+- under-replicated: observers will be promoted if the isr size drops below the configured count of replicas in the topic's replica placement policy.
+- leader-is-observer: observers will only be promoted if the leader is an observer.
```

Comment:
if the current leader is an observer, this can only happen after unclean leader election I believe and is the same as the previous behaviour. I've updated the wording to reflect this.